### PR TITLE
APU page now shows XX on apu shutdown

### PR DESCRIPTION
**Pull request by noga-aviator**
once the apu is shutdown and N% back to 0, the apu page shows XX on the gauge

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js [hunk 1]
```diff
@@ -182,6 +182,7 @@ var A320_Neo_LowerECAM_APU;
             }
             this.apuInactiveTimer = -1
             this.lastAPUMasterState = 0
+            this.apuShutingDown = false
```
Comment (michaeldiguiseppi) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:185]: typo, should be shutting with 2 't's.

Comment (noga-aviator) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:185]: yup, will modify

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js [hunk 2]
```diff
@@ -191,12 +191,12 @@ var A320_Neo_LowerECAM_APU;
             if ((currentAPUMasterState !== this.lastAPUMasterState) && currentAPUMasterState === 1) {
                 this.apuInactiveTimer = 3
                 this.lastAPUMasterState = currentAPUMasterState
-                this.apuShutingDown = false
+                this.apuShuttingDown = false
             }
             if ((currentAPUMasterState !== this.lastAPUMasterState) && currentAPUMasterState === 0) {
-                this.apuShutingDown = true
+                this.apuShutitngDown = true
```
Comment (michaeldiguiseppi) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:197]: #whoops.

Comment (noga-aviator) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:197]: lmao

Comment (noga-aviator) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js]: done

**Comment by noga-aviator** [comment]
can i merge now ?

**Pull request merged**